EnglishFilter/sections/specific.txt

```diff
@@ -421,6 +421,7 @@ speechnotes.co##div.ad
 indiatoday.in##.liveblog-header-ad
 smartprix.com##.dadow-box
 smartprix.com##.tanner-smpx
+windows10forums.com##div.message--post
```

Comment:
I'd use this `windows10forums.com##.block-body > div.message--post:not([data-author])`.